InfiniteScrollView.js

```diff
@@ -70,10 +89,22 @@ class InfiniteScrollView {
     }
 
     if (this._distanceFromEnd(event) < this.props.distanceToLoadMore) {
-      this.props.onLoadMore();
+      this._onLoadMore();
     }
   }
 
+  _onLoadMore() {
+    this.props.onLoadMoreAsync().catch((error) => {
```

Comment:
@ide - I can think of one case -- if we fire the loading event through some mechanism other than the InfiniteScrollView itself. But we're not doing that at all right now.

So should we track this through some local state rather than a prop? When the request is in flight, we set `state.isLoading` to `true` and when it finishes (via `.done`) we reset it, blocking other requests during that time.